README.md

```diff
@@ -1,13 +1,17 @@
 # Quire
 
-Developed by Getty, Quire is a new, multiformat publishing tool available for immediate access and use. Quire is in limited beta, © J. Paul Getty Trust, and not yet released as open-source software. **For a free license to use Quire, please complete [this form](http://bit.ly/quire-signup).**
+## About Quire 
 
-Striving toward becoming a fully open-source project by spring 2022, we encourage contributions from our community members. No matter what level of experience you have, we welcome all contributions, big and small.
+Developed by Getty, Quire is a digital publishing tool that can create dynamic publications in a variety of formats, including web, print, and e-book. Quire is an ideal tool for anyone seeking to publish beautiful, scholarly digital books optimized for visual imagery and designed to ensure their content is widely accessible and stable. 
```

Comment:
> ... and designed to ensure their content is widely accessible and stable.

Something about the wording here is awkward with the first half of the sentence. Perhaps to do with the "their" and whether that's referring to the "digital books", or to the "anyone seeking"?